### DISCUSSION TITLE
Dois Relacionamentos entre as mesmas tabelas

### DISCUSSION THREAD

**Discussion by danilobicas-unoeste** [balta-io/forum#600] [Tira Dúvidas] [answered]
Bom dia.
Tenho uma tabela/entidade Usuários  que armazena (funcionários e clientes) e preciso relacionar com a tabela/entidade Compras. Uma compra foi atendida por um funcionário e foi realizada para um cliente.... então eu tenho dois relacionamentos com a tabela de usuários e vice-versa. Como eu faço isso no mapeamento???

obrigado
Atte.
Danilo

**Comment by henriqueweege** [comment, marked as answer]
Fala, @danilobicas-unoeste ! Tudo certo?
Na tabela de Compras você vai precisar de duas colunas,  FuncionarioId e ClienteId, ambas como FK da tabela Usuários. 
Na hora de gravar uma compra, basta passar os Ids correspondentes.
Valeeeu!
